go-controller/Makefile

```diff
@@ -9,6 +9,12 @@ PKGS ?=
 GOPATH ?= $(shell go env GOPATH)
 TEST_REPORT_DIR?=$(CURDIR)/_artifacts
 export TEST_REPORT_DIR
+GO_DKR_IMG = golang:1.15.7-buster
```

Comment:
nit: `GO_DOCKER_IMG` instead of `GO_DKR_IMG`.
Also, what is the need for a `buster` image?

Reply:
As an aside, I ran into docker pull limits when pulling this image. Do we just assume that the buster image is hosted on a registry that is pullable in the env where this runs?

Reply:
@abhat, I am using the official go image from the docker registry, so most folks that have docker installed should be able to pull from the docker registry